_docs/vision.md

```diff
@@ -13,51 +13,40 @@ to present itself as more than just one database? Why not create software which
 is capable of speaking various protocols and presenting itself as more than just
 PostgreSQL?
 
-Welcome to Babelfish. The idea of the project is to close the gap between
-PostgreSQL and other systems by making PostgreSQL speak other protocols such as
-TDS, and maybe a lot more in the future. By implementing "MS SQL"-slang, we make
-it easier for people to move to PostgreSQL. 
+Welcome to Babelfish.  Babelfish closes the gap between PostgreSQL and
+Microsoft SQL Server by teaching PostgreSQL to speak TDS, the native
+client-server protocol of Microsoft SQL Server.  In addition, Babelfish
+implements support for Microsoft SQL Server syntax, which makes it easier to
+move to PostgreSQL.
 
 However, moving applications is just one side of the equation. Migrating data
 is one thing - making sure that applications work properly is another, far
 more important thing. In fact, migrating data to PostgreSQL can be relatively
-easy, while migration entire application stacks can be really difficult. 
+easy, while migrating entire application stacks can be really difficult.
 
 That's exactly why Babelfish is so important. It allows you to take existing
 applications, hook them up to TDS-enabled PostgreSQL and run them more quickly
 and more easily, even while some migration work might still be going on. 
 
-A second option is to move application from MS SQL to PostgreSQL without
-touching the application at all. Why is that so important? If you get your
-source code under control, life is usually easier. But what if you want to move
-old legacy applications from Microsoft SQL Server to PostgreSQL, legacy applications which cannot be
-modified easily, because the source code is not open, is lost, or is simply too hard to
-change? Babelfish is an excellent vehicle to help in this case as well. You can
-move away from expensive MS SQL deployments without doing a painful app
-migration. Brilliant, isn't it?
+If you are stuck with a legacy application, where the source code is not
```

Comment:
Correction: "If you are stuck with a legacy application where the source code is not available, or is too hard to modify for technical or financial reasons, Babelfish provides an easy way out."

Reply:
Done.